src/routes/revenue.route.ts

```diff
@@ -6,7 +6,7 @@ import { isAuthenticated } from '../middlewares/auth';
 export default class RevenueRoute {
   router = express.Router();
   revenueController = new RevenueController();
-  path = '/revenue';
+  path = '/v1/revenue';
```

Comment:
change all routes the way it was and add versioning to the main route in app.ts file.

Reply:
Thanks boss